modules/utils/docker/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #! /bin/bash
 
+COMMS_PCB_VERSION_FILE="/opt/hardware/comms_pcb_version"
```

Comment:
This is just for future tracking, we can combine all required config for the container later with different subnodes